src/app/main/component/shared/components/warning-pop-up/warning-pop-up.component.spec.ts

```diff
@@ -93,3 +96,6 @@ describe('WarningPopUpComponent', () => {
     });
   });
 });
+function getStepperFinal() {
+  throw new Error('Function not implemented.');
+}
```

Comment:
Why do you need this?

Reply:
it was self implemented. Forgot to delete!